src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -211,9 +221,11 @@ function ReportActionItemMessageEdit(props) {
     const deleteDraft = useCallback(() => {
         debouncedSaveDraft.cancel();
         Report.saveReportActionDraft(props.reportID, props.action.reportActionID, '');
-        ComposerActions.setShouldShowComposeInput(true);
-        ReportActionComposeFocusManager.clear();
-        ReportActionComposeFocusManager.focus();
+
+        if (isActive()) {
+            ReportActionComposeFocusManager.clear();
+            ReportActionComposeFocusManager.focus();
+        }
```

Comment:
I think this may be causing a regression. Can we remove the condition here? Currently if we delete a message the focus is not set back to the main composer but it should

Reply:
I think that solution won't work. We still need to find a way to fix the regression though

Reply:
I think we should focus the main composer if the deleted message is active. Please let me know if I am wrong

Reply:
That's correct but apparently this is not the behavior

https://github.com/Expensify/App/assets/16493223/b3b6c624-3a92-48da-9452-e417940e4755

Reply:
@s77rt 

There were some changes recently. This is due to [this PR](https://github.com/Expensify/App/pull/26741). Let me find a way

Reply:
I fixed the bug of `isActiveReportAction`. Now this works fine as expected

Reply:
I guess this is resolved, right?

Reply:
yes

cc @s77rt

Reply:
ok, thanks.